compiler/rustc_resolve/src/late.rs

```diff
@@ -1116,7 +1116,7 @@ impl<'a: 'ast, 'b, 'ast> LateResolutionVisitor<'a, 'b, 'ast> {
         }
     }
 
-    fn visit_generic_param_vec(&mut self, params: &'ast Vec<GenericParam>, add_self_upper: bool) {
+    fn visit_generic_param_slice(&mut self, params: &'ast [GenericParam], add_self_upper: bool) {
```

Comment:
maybe rename to "visit_generic_params" unless that name is already used?

Reply:
Sure, I don't see this name used.